src/System.Diagnostics.Tools/src/System.Diagnostics.Tools.csproj

```diff
@@ -5,7 +5,7 @@
     <ProjectGuid>{0B68298B-4672-4CA0-AD25-2F9ABEA1FF95}</ProjectGuid>
     <AssemblyName>System.Diagnostics.Tools</AssemblyName>
     <IsPartialFacadeAssembly>true</IsPartialFacadeAssembly>
-    <NuGetTargetMoniker Condition="'$(TargetGroup)' == ''">.NETStandard,Version=v1.3</NuGetTargetMoniker>
+    <NuGetTargetMoniker Condition="'$(TargetGroup)' == ''">.NETStandard,Version=v1.7</NuGetTargetMoniker>
```

Comment:
Note this was just bumped in https://github.com/dotnet/corefx/pull/11986. That PR will give you an indication of some of the things that change as part of the bump.